ArduCopter/mode_auto.cpp

```diff
@@ -771,33 +771,99 @@ void Copter::ModeAuto::wp_run()
         set_throttle_takeoff();
         return;
     }
-
-    // process pilot's yaw input
     float target_yaw_rate = 0;
-    if (!copter.failsafe.radio) {
-        // get pilot's desired yaw rate
-        target_yaw_rate = get_pilot_desired_yaw_rate(channel_yaw->get_control_in());
-        if (!is_zero(target_yaw_rate)) {
-            auto_yaw.set_mode(AUTO_YAW_HOLD);
-        }
-    }
 
-    // set motors to full range
+    //Enable course locked add a2sandres

```

Comment:
Please add space after `//` and remove `add a2sAndres`.

Reply:
revised and changed